signer/Cargo.toml

```diff
@@ -33,7 +33,7 @@ std = [
 # https://github.com/rust-bitcoin/rust-bitcoin/issues/930#issuecomment-1215538699
 sr25519 = ["schnorrkel"]
 ecdsa = ["secp256k1"]
-unstable-eth = ["keccak-hash", "ecdsa", "secp256k1", "bip32"]
+unstable-eth = ["dep:subxt-core", "keccak-hash", "ecdsa", "secp256k1", "bip32"]
```

Comment:
do we need the keccak-hash now when it's dependency in subxt-core?

Reply:
From my prev comment, let's remove the subxt-core dep here :)